app/src/main/java/org/sopt/carrot/presentation/productDetailScreen/components/ProductBottomBar.kt

```diff
@@ -0,0 +1,177 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.VerticalDivider
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.carrot.R
+import org.sopt.carrot.core.extension.noRippleClickable
+import org.sopt.carrot.ui.theme.CarrotTheme
+
+
+@Composable
+private fun PriceInfo(
+    productPrice: String,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(4.dp),
+        modifier = modifier
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(8.dp)
+        ) {
+            Text(
+                text = "${productPrice}원",
+                style = CarrotTheme.typography.body.b_16_08,
+                color = CarrotTheme.colors.gray8
+            )
+            VerticalDivider(
+                modifier = Modifier.height(12.dp),
+                thickness = 1.dp,
+                color = CarrotTheme.colors.gray3
+            )
+            Image(
+                painter = painterResource(id = R.drawable.img_daangnpay),
+                contentDescription = "당근페이 이미지"
+            )
+        }
+        Text(
+            text = "가격 제안 불가",
+            color = CarrotTheme.colors.gray6,
+            style = CarrotTheme.typography.body.b_14
+        )
+    }
+}
+
+@Composable
+private fun LikeButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    var isLiked by remember { mutableStateOf(false) }
+
+    Icon(
+        modifier = modifier
+            .size(28.dp)
+            .noRippleClickable {
+                isLiked = !isLiked
+                onClick()
+            },
+        imageVector = if (isLiked) {
+            ImageVector.vectorResource(id = R.drawable.ic_like_lg_clicked)
+        } else {
+            ImageVector.vectorResource(id = R.drawable.ic_like_lg_28)
+        },
+        contentDescription = "찜하기",
+        tint = if (isLiked) {
+            Color(0xFFFF4F4F) // 빨간색으로 설정
```

Comment:
p3: 이렇게 하면 주석 없어도 이해가 될거 같습니다!! (그냥 간단한 다른방법도 있다 느낌?)
```suggestion
            Color.Red
```

Reply:
적용했습니다!